doc/usage/configuration.rst

```diff
@@ -2483,5 +2483,5 @@ Options for the C++ domain
 Example of configuration file
 =============================
 
-.. literalinclude:: _static/conf.py.txt
+.. literalinclude:: ../_static/conf.py.txt
```

Comment:
Please use `/_static/conf.py.txt` instead. A path starting with `/` is considered as relative path from srcdir. It is much simpler.